### ISSUE TITLE
Disable language folders

### ISSUE THREAD

**Issue by stumi01** [lokalise/lokalise-cli-2-go#2]
We are trying to migrate from the cli v1 and having trouble with the language folders.

We found a workaround to unzip to the parent directory and set the `directory-prefix` to the project dir but its a little inconvenient  

v1 command was:
```   
./lokalise --token "${LOKALISE_TOKEN}" export "${LOCALISE_PROJECT}" --type xml --use_original 1 --no_language_folders 1 --unzip_to .

```
v2 command:
```
./lokalise2 file download --token "${LOKALISE_TOKEN}" --project-id "${LOCALISE_PROJECT}" --format xml --original-filenames true --directory-prefix "android_app_dir" --unzip-to "../"
```

**Comment by nickustinov** [comment]
@stumi01 That was a legacy parameter from API1. It's similar to specifying --directory-prefix=""

**Comment by stumi01** [comment]
Using the --directory-prefix "" seems to have no effect. It recreates the language dirs

**Comment by nickustinov** [comment]
Got it! Will fix in 2.03, but as you have a workaround it's coming next week probably. Let us know if you come across any other anomalies and thanks for your input!

**Comment by nickustinov** [comment]
@stumi01 2.03 released, give it a try. Should pass empty parameter now.
